__tests__/integration.test.js

```diff
@@ -0,0 +1,32 @@
+const request = require("supertest")
+const db = require("../db/connection")
+const seed = require("../db/seeds/seed")
+const data = require("../db/data/test-data/index");
+const app = require("../app")
+
+beforeEach(() => seed(data));
+afterAll(() => db.end())
+
+describe("GET: /api/topics", () => {
+    test("GET: responds with a 200 status code and an array of topic objects", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+            expect(body.topics.length).toBe(3)
```

Comment:
Great that you're checking the length of the response to avoid false positives if the array turns out empty 👍